content/en/news/releases/1.19.x/announcing-1.19/change-notes/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Istio 1.19.0 Change Notes
+linktitle: 1.19.0
+subtitle: Minor Release
+description: Istio 1.19.0 change notes.
+publishdate: 2023-09-01
+release: 1.19.0
+weight: 20
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+- There are no new deprecations in Istio 1.19.0
```

Comment:
We do announce that we are going to remove certain build artifacts in a future release, see the item under Installation. It sounds like it should be copied/moved up here.

Reply:
Ah yes, good callout. I think the changed packages for Windows and macOS should go here. Though not necessarily deprecated functionality, it is a deprecation of sorts that will break some automation.

Reply:
I first thought of this as the place were we list those things that we deprecated earlier and are finally removing then, but looking at the first sentence: `These notices describe functionality that will be removed in a future release according` I figure this points to items that we will remove later. I saw at least one item where we stated what was deprecated earlier is now removed. I suspect we will see that in a few releases for this item.